packages/components/src/box-control/index.js

```diff
@@ -51,13 +51,22 @@ export default function BoxControl( {
 	label = __( 'Box Control' ),
 	values: valuesProp,
 	units,
+	sides,
+	resetToInitialValues = false,
```

Comment:
not sure I understand this new prop? What do we need it for?

Reply:
I found within the Global Styles sidebar panel the behaviour of the BoxControl's reset button was different. 

When used in the context of block support, it makes sense for the reset button to set the values to the empty undefined state it was previously. However, within Global Styles it seemed to make more sense for the reset button to reset the values to what was initially retrieved from the theme.json etc. This new prop was to indicate that situation as opposed to simply having custom values set previously by the user.

Reply:
Mmmm makes sense but the current implementation might be a bit confusing/buggy. I think it relies on the "value" prop when first mounted which can be different from the value that comes from `theme.json` (say a user edited, saved and reload the page). So maybe a better solution would be to actually pass a "resetValue" prop.

Reply:
That was my intention to allow the user to reset it to what they started with on loading the editor whether that be their previous saved values or what was coming from the theme.json.

Would it be preferred to only reset back to the theme.json's values?

Reply:
I've implemented the suggested change to pass reset values to the `BoxControl`. It now also retrieves the original style values from the theme.json and will reset to them. 

If we decide we do want the user to be able to reset to their previously saved values a ref to the original values can be added to the spacing panel and passed through the same way as the theme values now.